go.mod

```diff
@@ -1,15 +1,18 @@
 module github.com/kubev2v/migration-planner
 
-go 1.21.11
+go 1.22.2
```

Comment:
Is this part of `registry.access.redhat.com/ubi9/go-toolset` already?

Reply:
not me. maybe go mod.